src/networking/responses/stripe-elements.ts

```diff
@@ -16,8 +16,45 @@ export interface StripeElementsConfiguration {
   currency?: string;
 }
 
+export interface StripeExpressCheckoutOptions {
+  applePay?: {
+    recurringPaymentRequest?: {
+      paymentDescription: string;
+      managementURL: string;
+      regularBilling: {
+        amount: number;
+        label: string;
+        recurringPaymentStartDate?: Date;
+        recurringPaymentEndDate?: Date;
+        recurringPaymentIntervalUnit?:
+          | "year"
+          | "month"
+          | "day"
+          | "hour"
+          | "minute";
+        recurringPaymentIntervalCount?: number;
+      };
+      trialBilling?: {
+        amount: number;
+        label?: string;
+        recurringPaymentStartDate?: Date;
+        recurringPaymentEndDate?: Date;
+        recurringPaymentIntervalUnit?:
+          | "year"
+          | "month"
+          | "day"
+          | "hour"
+          | "minute";
+        recurringPaymentIntervalCount?: number;
+      };
+      billingAgreement?: string;
+    };
+  };
+}
+
 export interface GatewayParams {
   stripe_account_id?: string;
   publishable_api_key?: string;
   elements_configuration?: StripeElementsConfiguration;
+  express_checkout_configuration?: StripeElementsConfiguration;
```

Comment:
We'll need to also add this one as optional parameter in the `CheckoutCalculateTaxResponse`